blenderproc/python/object/ObjectMerging.py

```diff
@@ -1,10 +1,14 @@
+"""Merges several different objects"""
+
 from typing import List
 
 from blenderproc.python.types.EntityUtility import Entity, create_empty
 
 
 def merge_objects(objects: List[Entity], merged_object_name: str = 'merged_object') -> Entity:
-    """ Generates an empty object and sets this as parent object for all objects in the list which do not already have a parent set.
+    """
```

Comment:
Why?

Reply:
Because the summary description should be stated in the first line. I mean, we also do this everywhere else in our code

See:
https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html#template-py-source-file
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
Good :)